File: admin/web/src/api/dayz/dayzitem.js
import service from '@/utils/request'

export const createDayzItem = (data) => {
  return service({
    url: '/dayzItem/createDayzItem',
    method: 'post',
    data,
  })
}

export const updateDayzItem = (data) => {
  return service({
    url: '/dayzItem/updateDayzItem',
    method: 'put',
    data,
  })
}

export const deleteDayzItemByIds = (data) => {
  return service({
    url: '/dayzItem/deleteDayzItemByIds',
    method: 'delete',
    data,
  })
}

export const getDayzItem = (data) => {
  return service({
    url: '/dayzItem/getDayzItem',
    method: 'post',
    data,
  })
}

export const getDayzItemByID = (data) => {
  return service({
    url: '/dayzItem/getDayzItemByID',
    method: 'get',
    data,
  })
}

export const getDayzItemList = (data) => {
  return service({
    url: '/dayzItem/getDayzItemList',
    method: 'post',
    data,
  })
}

File: admin/server/router/dayz/dayz_item.go
package dayz

import (
	v1 "github.com/flipped-aurora/gin-vue-admin/server/api/v1"
	"github.com/gin-gonic/gin"
)

type DayzItemRouter struct{}

func (e *DayzItemRouter) InitDayzItemRouter(Router *gin.RouterGroup) {
	dayzItemRouter := Router.Group("dayzItem")
	dayzItemApi := v1.ApiGroupApp.DayzApiGroup.DayzItemApi
	{
		dayzItemRouter.POST("createDayzItem", dayzItemApi.CreateDayzItem)
		dayzItemRouter.PUT("updateDayzItem", dayzItemApi.UpdateDayzItem)
		dayzItemRouter.DELETE("deleteDayzItemByIds", dayzItemApi.DeleteDayzItemByIds)
	}
	{
		dayzItemRouter.POST("getDayzItem", dayzItemApi.GetDayzItem)
		dayzItemRouter.GET("getDayzItemByID", dayzItemApi.GetDayzItemByID)
		dayzItemRouter.POST("getDayzItemList", dayzItemApi.GetDayzItemList)
	}
}
